CeCILL-C.txt

```diff
@@ -0,0 +1,14 @@
+Copyright (C) 2006-2015 INRIA and contributors
```

Comment:
What about CeCILL-C-long.txt
<https://github.com/INRIA/spoon/pull/449#discussion_r46989343> and
CeCILL-C-short.txt
<https://github.com/INRIA/spoon/pull/449#discussion_r46989343> and
LICENSE.txt points to CeCILL-C-long.txt?
<https://github.com/INRIA/spoon/pull/449#discussion_r46989343>